javascript/chronometer.js

```diff
@@ -1,34 +1,44 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(callback) {
-    // ... your code goes here
+    this.intervalId = setInterval(() => {
+      this.currentTime++;
+      callback();
+    }, 10)
   }
```

Comment:
Here you should be checking if the argument is passed into `start()` method. To be sure app does not crash or throw error

`if (callback) { callback() }`